packages/connection-form/src/components/connection-form.tsx

```diff
@@ -521,6 +521,17 @@ function ConnectionForm({
                 </div>
               </Banner>
             )}
+            {protectConnectionStrings && (
```

Comment:
I know it's a corner case, but would it make more sense to make this banner show up only when you can actually edit the connection string taking connection status into account? Otherwise it's getting pretty packed in the when editing a connected one:

![image](https://github.com/user-attachments/assets/c121d46f-a472-45ad-a149-d769a7fca5fd)

Reply:
Aah well spotted. Fixed.